truss/templates/server/model_wrapper.py

```diff
@@ -617,7 +624,7 @@ async def _stream_with_background_task(
         generator: Union[Generator[bytes, None, None], AsyncGenerator[bytes, None]],
         span: trace.Span,
         trace_ctx: trace.Context,
-        release_and_end: Callable[[], None],
+        release_and_end: Callable[[], None] = lambda: None,
```

Comment:
We intentionally don't want to reuse the `predict` semaphore for these endpoints, and we want to encourage users to build concurrency controls in their model code if they need it

A great future goal could be a `@truss.concurrency(max_requests = 2)` or something